src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/MarshallingAttributeInfo.cs

```diff
@@ -16,7 +16,8 @@ namespace Microsoft.Interop
     /// Type used to pass on default marshalling details.
```

Comment:
Can we add a comment here about how we can't use this type as a field in any type derived from `MarshallingInfo`? We need to be careful to not include any `ISymbol`-derived types in any `MarshallingInfo`-derived types, otherwise we'll break incremental support.

We should also add an incremental test to ensure that we don't break this in the future accidentally, but that can wait until https://github.com/dotnet/runtime/pull/63978 is in.